modules/core/src/main/scala/org/scalasteward/core/io/WorkspaceAlg.scala

```diff
@@ -40,24 +40,22 @@ object WorkspaceAlg {
       F: FlatMap[F]
   ): WorkspaceAlg[F] =
     new WorkspaceAlg[F] {
-      private[this] val reposDir = config.workspace / "repos"
+      private val reposDir: File =
+        config.workspace / "repos"
 
-      private[this] def repoDirUnsafe(repo: Repo) = reposDir / repo.owner / repo.repo
+      private def mkRepoDir(repo: Repo): File =
```

Comment:
only a comment about naming :-)
I first thought that `mkRepoDir` creates the directory.
Suggestion:
* `val reposDir` => `val reposRootDir`
* `def mkRepoDir(repo: Repo)` => `def repoDir(repo: Repo)`

If I got the meaning right

Reply:
> I first thought that mkRepoDir creates the directory.

Right, this is a bad name. But we can't use `repoDir` since this already exists. I tend to rename `mkRepoDir` to `toFile`.

Reply:
looks fine, too